### PR TITLE
Us 04

### PR DESCRIPTION
**Pull request by NicosMQ**

### CODE REVIEW

**Comment by JyAjAn** [comment]
ok

**Review by JyAjAn — APPROVED** [review]
good

File: client/src/pages/Inscription/Inscription.jsx [hunk 1]
```diff
@@ -1,10 +1,92 @@
+import { useRef } from "react";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```
Comment (Alexandre78R) [inline client/src/pages/Inscription/Inscription.jsx]: Delete and add this line in App.css or App.jsx

File: server/package.json
Comment (Alexandre78R) [inline server/package.json:1]: remove axios from package.json

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by Ludovicscelles — APPROVED** [review]

**Review by Alexandre78R — APPROVED** [review]
Good

**Pull request merged**